include/battle.h

```diff
@@ -12,6 +12,8 @@
 #include "battle_util2.h"
 #include "battle_bg.h"
 
+#include "pokeball.h" // i'm not sure about this one
```

Comment:
i'm unsure about including this header here does anyone have anything to say about this?

Reply:
It's fine, but group it with the other headers and remove the comment